packages/ozone/src/mod-service/views.ts

```diff
@@ -224,24 +224,33 @@ export class ModerationViews {
   }
 
   async repoDetail(
-    did: string,
+    dids: string | string[],
     labelers?: ParsedLabelers,
-  ): Promise<RepoViewDetail | undefined> {
+  ): Promise<Map<string, RepoView>> {
+    const results = new Map<string, RepoView>()
+    const didList = Array.isArray(dids) ? dids : [dids]
     const [repos, localLabels, externalLabels] = await Promise.all([
-      this.repos([did]),
-      this.labels(did),
-      this.getExternalLabels([did], labelers),
+      this.repos(didList),
+      this.labels(didList),
+      this.getExternalLabels(didList, labelers),
     ])
-    const repo = repos.get(did)
-    if (!repo) return
 
-    return {
-      ...repo,
-      moderation: {
-        ...repo.moderation,
-      },
-      labels: [...localLabels, ...externalLabels],
-    }
+    repos.forEach((repo, did) => {
+      const labels = [
+        ...(localLabels.get(did) || []),
+        ...externalLabels.filter((label) => label.uri === did),
```

Comment:
I wonder if `getExternalLabels` should return a `Map<string, Label[]>` instead of just a big unfiltered array. That way we don't need to repeatedly scan over the array to get our list of labels for each subject